lib/takeaway.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'menu'
+
+class Takeaway
+  attr_reader :order
+
+  @@menu = Menu.new
```

Comment:
At the moment, the Menu is hard coded so all Takeaways will have the same Menu.  Can you think of how to make this more flexible such that each Takeaway you initialize can have a different Menu?